releases/24.09.0/24090 Release Notes.md

```diff
@@ -0,0 +1,469 @@
+# O3DE 24.09.0 Release Notes
+
+Get ready for a game-changing update with the O3DE 24.09 release. This latest version is packed with fantastic enhancements driven by your feedback, focusing on turbocharging performance and elevating your experience. Now, with the installer version, you can build projects using only Script Canvas and Lua—no compiler needed\! But that’s just the beginning. We’ve supercharged editor startup times with up to a jaw-dropping 90% boost for large projects brimming with assets. Mobile developers, rejoice\! Enjoy a staggering 400% increase in performance on both iOS and Android, along with dramatically reduced runtime memory usage. For simulation use cases, we have added the Georeference component, ability to parse Gazebo data, and a ROS2FrameComponent, along with many simulation performance improvements. This release is bursting with hundreds of quality of life improvements and bug fixes. Dive in to explore just a few of the exciting highlights that 24.09 has in store.
+
+See Simulation, Rendering, and AR/VR sections for specific changes in these areas.
+
+## General Changes and Improvements in 24.09.0
+
+**Major Improvements:**
+
+* Quick-Start “Script Only” Projects \- Can build projects using Script Canvas and Lua, without needing a C++ compiler,  using the O3DE installer version.  
+* Many optimizations for memory usage on mobile and AR/VR devices  
+* Mobile performance \- up to 400% improvement across iOS, Android and AR/VR/XR devices  
+* Split PhysX 4 and 5 into separate gems, enabling users to switch between the PhysX versions easily, even when using installer builds.  
+* Added a Project Export UI along wth full export support for iOS, Android, Linux, and Windows. Users are no longer required to use a command line.  
+* Added options to reduce size of headless server up to 90%, dependent on assets used.   
+* Added a mobile specific rendering pipeline, allowing users to easily enable/disable features as needed.  
+* Added ability to control quality settings per device. There is a default of 3 performance levels (low, medium, high) based on device specifications for CPU, GPU, and memory.  
+* Added shader variants. When setup, this allows the renderer to automatically utilize the most performant shader for a given rendering need.   
```

Comment:
I think this needs rewording, because as described, this feature has been available for a long time now? If this trying to describe Specialization Constants? or improvements in the ShaderManagementConsole?